src/components/WhimsicalInstaller/WhimsicalInstaller.js

```diff
@@ -68,10 +68,12 @@ class WhimsicalInstaller extends PureComponent<Props, State> {
 
   toggleRunning = () => {
     if (this.props.isRunning) {
-      this.wrapperBoundingBox = this.wrapperNode.getBoundingClientRect();
+      if (this.wrapperNode) {
```

Comment:
I think we can roll this into the previous condition:

```js
if (this.props.isRunning && this.wrapperNode) {
```

I like reducing nested conditionals, when possible.